frontend/packages/console-app/src/components/guided-tours/GuidedTourCatalog.scss

```diff
@@ -0,0 +1,44 @@
+.odc-guided-tour {
+  &__catalog {
+    background: var(--pf-global--palette--black-150);
+    padding: var(--pf-global--spacer--md);
+    height: 100%;
+  }
+  &__status {
+    margin: var(--pf-global--spacer--sm) 0;
+  }
+  &__icon {
+    width: var(--pf-global--spacer--lg);
+    height: var(--pf-global--spacer--lg);
+    margin: 0 var(--pf-global--spacer--sm) 0 0;
+  }
+  &__description {
+    height: 4rem;
```

Comment:
Fixed heights are bad here. Perhaps a min height, but i think the tiles will need to scale. Unless UX imposes a fixed height and then does a cut off for the text (white gradient fade).